src/index.md

```diff
@@ -8,6 +8,19 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+save your website's files locally, on your users' devices. Consider Workbox if
+you want to:
+
```

Comment:
Looks good. I've seen `store` and `storage` used more often when describing the action of offline-caching data. I wonder if that would make more sense here than `save`? Your call.